### PR TITLE
DOCS-2281 Re-write API Abuse Prevention

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/about-wallarm/api-abuse-prevention.md [hunk 1]
```diff
@@ -39,21 +39,6 @@ If one or several detectors point to [bot attack signs](#automated-threats-block
 
 The solution deeply observes traffic anomalies before attributing them as malicious bot actions and blocking their origins. Since metric collection and analysis take some time, the module does not block malicious bots in real-time once the first malicious request originated but significantly reduces abnormal activity on average.
 
-## Activating API Abuse Prevention
-
-The **API Abuse Prevention** module in the disabled state is delivered with [all forms of the Wallarm node 4.2 and above](../installation/supported-deployment-options.md) including the CDN node.
-
-To activate API Abuse Prevention:
-
-1. Make sure that your traffic is filtered by the Wallarm node 4.2 or later.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-abuse-prevention.md:48]: delete

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+## Enabling
 
-* **Administrator** or **Analyst** for the regular accounts.
-* **Global Administrator** or **Global Analyst** for the accounts with the multitenancy feature.
+The API Abuse Prevention module in the disabled state is delivered with [all forms of the Wallarm node 4.2 and above](../installation/supported-deployment-options.md) including the CDN node.
 
-## Creating API abuse profile
+To enable API Abuse Prevention:
+
+1. Make sure that your traffic is filtered by the Wallarm node 4.2 or later.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:11]: delete

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+1. Make sure your [subscription plan](subscription-plans.md#subscription-plans) includes **API Abuse Prevention**. To change the subscription plan, please send a request to [sales@wallarm.com](mailto:sales@wallarm.com).
+1. In Wallarm Console → **API Abuse Prevention**, create or enable at least one [API Abuse profile](../user-guides/api-abuse-prevention.md).
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:13]: please chek this link and all other links - it does not work if you just copy content from one file to another

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
+This article describes how to enable and configure the [API Abuse Prevention](../about-wallarm/api-abuse-prevention.md) module to detect and mitigate malicious bots and to avoid blocking legitimate activities.
 
-The section is only available to the users of the following [roles](../user-guides/settings/users.md#user-roles):
+## Enabling
 
-* **Administrator** or **Analyst** for the regular accounts.
-* **Global Administrator** or **Global Analyst** for the accounts with the multitenancy feature.
+The API Abuse Prevention module in the disabled state is delivered with [all forms of the Wallarm node 4.2 and above](../installation/supported-deployment-options.md) including the CDN node.
 
-## Creating API abuse profile
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:10]: why not to merge the sections about enabling and configuring profiles?

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 5]
```diff
@@ -1,13 +1,25 @@
-# API Abuse Prevention profile management <a href="../../about-wallarm/subscription-plans/#subscription-plans"><img src="../../images/api-security-tag.svg" style="border: none;"></a>
+# API Abuse Prevention Setup <a href="../../about-wallarm/subscription-plans/#subscription-plans"><img src="../../images/api-security-tag.svg" style="border: none;"></a>
 
-In the **API Abuse Prevention** section of Wallarm Console you can manage API abuse profiles that are required for configuration of the [**API Abuse Prevention**](../about-wallarm/api-abuse-prevention.md) module.
+This article describes how to enable and configure the [API Abuse Prevention](../about-wallarm/api-abuse-prevention.md) module to detect and mitigate malicious bots and to avoid blocking legitimate activities.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:3]: the overview must point to the method of the configuration (profiles)

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 2 above)
Comment (AnastasiaTWW) [reply docs/latest/user-guides/api-abuse-prevention.md:11]: Wallarm node 4.2 and subscription plan are requirements

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 6]
```diff
@@ -23,18 +35,14 @@ To create an API abuse profile:
...
-
-## Deleting API abuse profile
-
 Deleted profiles are the ones that cannot be restored and that the **API Abuse Prevention** module does not use during traffic analysis.
 
-You can delete the profile by using the corresponding **Delete** option.
-
+You can find **Disable** and **Delete** options in the profile menu.
+ 
 ## Exploring blocked malicious bots and their attacks
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:46]: should not it be reflected in the article about exploration?

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 2 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-abuse-prevention.md:11]: The "Enabling" sections of other modules are written in the same way (for example, API Discovery, Credential Stuffing).

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 4 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-abuse-prevention.md:10]: The reason for not doing that relates to the comment about "Enabling" section: it now includes information about node version and subscription, this is not related to configuring profiles and in the same time cannot be deleted from "Enabling".

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 5 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-abuse-prevention.md:3]: It does:

> During the [API abuse profile setup](http://127.0.0.1:8000/user-guides/api-abuse-prevention/#creating-profiles), you can configure the API Abuse Prevention module to protect from all types of bots or limit protection only for specific threats.

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 6 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-abuse-prevention.md:46]: I forgot to delete it. Now fixed.

**Pull request merged**